build.gradle

```diff
@@ -95,6 +95,7 @@ ext {
   threeTenAbpAndroid = "com.jakewharton.threetenabp:threetenabp:$threeTenAbpVersion"
   picasso = "com.squareup.picasso:picasso:2.71828"
   ktx = "androidx.core:core-ktx:1.8.0"
+  kotlinReflection = "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
```

Comment:
Any way we can do this without Kotlin reflection?

Reply:
It doesn't really work as Dp is an inline class and java reflection can't really work with that and treats it as a float